code/test/Templates.Test/CodeStyleEnforcementTests.cs

```diff
@@ -102,6 +102,20 @@ void EnsureUwpTemplatesNotUsed(string fileExtension)
             EnsureUwpTemplatesNotUsed("*.vb");
         }
 
+        [Fact]
+        public void EnsureCodeDoesNotUseInvalidTodoCommentIdentifier()
+        {
+            void EnsureUwpTemplatesNotUsed(string fileExtension)
```

Comment:
This method name is not indicative of what it does

Reply:
How about EnsureCodeDoesNotUseSwappedTodoComment ?
I want to test that I do not accidentially swap TODO and WTS.

Reply:
Grrrr, GitHub interface. My comment was against line 108. "EnsureUwpTemplatesNotUsed" is what isn't appropriate

Reply:
ah, ok, now I see it, changed!